isis/src/base/apps/caminfo/caminfo.cpp

```diff
@@ -286,6 +286,67 @@ namespace Isis{
           camstats->append(MakePair("ObliqueResolutionMaximum", cg["ObliqueResolutionMaximum"][0]));
         }
 
+        // Extracts camstat data from existing CameraStatistics Table in cube label
+        if(ui.GetBoolean("USECAMSTATSTBL")) {
+
+          if (!incube->hasTable("CameraStatistics")) {
```

Comment:
In this case, I simply ran camstats even if the user specified to use the table in the label and it doesn't exist. In other words, if the table doesn't exist, create it by running camstats (they asked for it) and use the same process to extract all the keywords in the table (after correcting the typo for AspectRatioMinimum. (BTW, isn't this API breaking since the name changed even tho its a typo?)

Reply:
This falls into the grey area for API breaking. It changes the name of an output value, but we don't specify that as part of the API.

Something we should consider in this PR is for this to work with old camstats tables.

Personally, I think this should go out in a feature release with a note in the changelog and the app history.